tool-nextjs-starter/src/hooks/getContext.ts

```diff
@@ -0,0 +1,34 @@
+import {APP_ORIGIN, TOOL_ID} from "@/hooks/useAutoHeight";
+import {useCallback, useEffect, useState} from "react";
+
+
+//DRAFT
+export function useContext() {
+  const [context, setContext] = useState(undefined)
+
```

Comment:
Maybe `useToolContext()`? And what would be the difference between context and getContext?

Reply:
also thinking about useStoryContext.

actually in the beginning i thought that it would get the user more control and it would somehow follow the react hook pattern of returning the value and the function but now that i think of it should be more like this 

`const context = useContext()`